docs/state-diff/README.md

```diff
@@ -0,0 +1,185 @@
+ 🤖⚡ External Computation with Kontrol ⚡🤖
+=================================
+**Injecting Foundry Execution into Proofs**
```

Comment:
```suggestion
**Injecting Foundry Execution into Proofs for Improved Performance**
```
That's a draft suggestion, but I think it might be helpful to incorporate some information about why would we want to do it here, e.g. say that it's helpful for projects with complex but concrete setup process.

Reply:
I added reference to the usability in the subtitle, but since the readme is long enough, I thought to defer the motivation of why would anyone endure these steps to the blog post. Does that sound alright?

Reply:
Yes, agreed!